storybook/src/components/Radio/Radio.stories.tsx

```diff
@@ -71,4 +71,9 @@ export const RadioGroup: Story = {
       </Radio>
     </Fieldset>
   ),
+  parameters: {
+    docs: {
+      source: { type: 'dynamic' },
```

Comment:
Renders the story source with dynamically updated arg values – https://storybook.js.org/docs/api/doc-block-source#type